### ISSUE TITLE
No read permissions in gem

### ISSUE THREAD

**Issue by ChrisBr** [NeverMin/omniauth-wechat-oauth2#32]
As mentioned in https://github.com/NeverMin/omniauth-wechat-oauth2/pull/31, we had issues in production to require the gem. After using the new release, the bug was still not fixed however although our fork worked. After more debugging, we found out that the packaged gem does not have read permissions set except for privileged users. 

```
tar -tvf data.tar.gz
-rw-------  0 wheel  wheel    3720 11 Jun 08:46 README.md
-rw-------  0 wheel  wheel      26 11 Jun 08:46 lib/omniauth-wechat-oauth2.rb
-rw-------  0 wheel  wheel    2207 11 Jun 08:46 lib/omniauth/strategies/wechat.rb
-rw-------  0 wheel  wheel    2577 11 Jun 08:46 lib/omniauth/strategies/wechat_qiye.rb
-rw-------  0 wheel  wheel      79 11 Jun 08:46 lib/omniauth/wechat.rb
```

Seems something is wrong on the machine which is used to build and package the gem. Could you please fix the permission and release a new version. Thank you!

**Comment by Eric-Guo** [comment]
Yes, I set the [umask 077](https://github.com/drduh/macOS-Security-and-Privacy-Guide/issues/321) a long time ago, so lacking read permit for everyone and group probably should think as a feature instead of a *bug*.

Since it already impacts the production, I just release 0.2.2 which contains the everyone can having read permitted.

But I think it is still arguable here, and I believe original only user can read is the correct setting.
